src/bootstrap/builder.rs

```diff
@@ -1543,6 +1543,12 @@ impl<'a> Builder<'a> {
             }
         }
 
+        // For now --generate-link-to-definition is only available on nightly
+        // TODO(SkiFire13): Remove the stage > 0 check when beta reaches 1.56
```

Comment:
This should be done by whoever promotes the beta compiler :)
```suggestion
        // cfg(not(bootstrap)), remove the stage > 0 check when beta reaches 1.56
```